api/scpca_portal/test/test_utils.py

```diff
@@ -200,5 +200,5 @@ def test_get_csv_zipped_values_keys_not_present(self):
         }
         args = ["country", "language", "population", "currency"]
 
-        with self.assertRaises(AttributeError):
+        with self.assertRaises(ValueError):
```

Comment:
I assume we expect an error when keys are missing. The test was failing due to the behavior of the `zip` method (it stops when the shortest list ends) which results in `ValueError`. Thus, I updated the expected error type to `ValueError`.